### PR TITLE
added license-solver

### PR DESCRIPTION
**Pull request by meile18**
## Related Issues and Dependencies
 HOLD until [#26](https://github.com/thoth-station/license-solver/pull/26) and [#44](https://github.com/thoth-station/license-solver/pull/44)

## This introduces a breaking change

- [x] No

## This Pull Request implements

implement license-solver

### CODE REVIEW

File: thoth/solver/python/python.py [hunk 1]
```diff
@@ -317,6 +318,10 @@ def _do_resolve_index(python_bin, solver, all_solvers, requirements, exclude_pac
             )
             continue
 
+        # license solver
+        extracted_metadata["package_license"] = detect_license(extracted_metadata["importlib_metadata"]["metadata"])
+        print(extracted_metadata["package_license"])
```
Comment (harshad16) [inline thoth/solver/python/python.py:323]: seems like debugging print is missed here
can you please change it to logging, if we should have it in source code.

**Review by harshad16 — CHANGES_REQUESTED** [review]
lgtm
minor change needed

File: thoth/solver/python/python.py (hunk 1 above)
Comment (meile18) [reply thoth/solver/python/python.py:323]: Ohh sorry, my bad. I forgot to change it after debugging :smiling_face_with_tear:

**Comment by goern** [comment]
what is the status of this one?

**Comment by meile18** [comment]
we are waiting for the license-solver release on PyPI and check of thoth-storage if it's all okay.

**Comment by meile18** [comment]
do not merge! license-solver has changed output [#44](https://github.com/thoth-station/license-solver/pull/44)

**Review by fridex — COMMENTED** [review]
Changes look good to me, I will also try this. @meile18 do we want a new thoth-license-solver release?

**Comment by meile18** [comment]
yes @fridex, the new release contains changed output

**Comment by goern** [comment]
is this ready to merge?

**Review by fridex — COMMENTED** [review]
changes look good to me, @meile18 is the desired `detect_license` ~~login~~ logic available in thoth-license-solver available on PyPI (I see version 0.1.4)?

**Comment by meile18** [comment]
Hi @fridex @goern. Yes, this is ready to merge.

**Comment by fridex** [comment]
Perfect, thanks! 👍🏻 💯

**Pull request merged**